config.toml

```diff
@@ -0,0 +1,133 @@
+ResourceFilePath = "resources/"
+
+[DataService]
+RemoteEndpoint = "http://127.0.0.1:11111"
+BindIP = "0.0.0.0"
+Port = 11111.0
```

Comment:
.0 ???

Reply:
Ops... I used a tool to convert our json to toml, some mistakes in there...